doc/docs.md

```diff
@@ -1,5 +1,7 @@
 # V Documentation
 
+## Translation
+[Portuguese](pt_br/docs.md)
```

Comment:
I am not sure that adding alternative translation links at the start of the docs.md is the best course of action at this stage, although docs.md is pretty stable, it may be hard to keep/check several translations up to date with it 🤔 .
@medvednikov what do you think?

Reply:
I got your point. So, how about to put a link to /pt_br/docs.md on vlang.io?